lib/index.js

```diff
@@ -95,13 +100,14 @@ const runTasks = async (opts, di) => {
       version = version || (await reduceUntil(plugins, plugin => plugin.getIncrementedVersion(incrementBase)));
     }
 
-    if (!version) {
-      log.obtrusive(`No new version to release`);
-    }
-
     if (isReleaseVersion) {
-      console.log(version);
-      process.exit(0);
+      if (version) {
+        console.log(version);
```

Comment:
While I like the idea and result of this all, here's a bit of a nitpick: why not use the `Log` class here anymore? In my mind, the `Log` class for logging, to abstract away `stdout` vs `stderr` and no need to use `console.*` directly.

Reply:
Well, it is not logging, its direct output of the command. But it does not really matter, I did the changes incrementally, and with final version, its not needed to modify this file anymore.